mesonbuild/backend/backends.py

```diff
@@ -1400,6 +1400,15 @@ def get_build_by_default_targets(self) -> 'T.OrderedDict[str, T.Union[build.Buil
                 result[name] = b
         return result
 
+    def get_install_targets(self) -> 'T.OrderedDict[str, T.Union[build.BuildTarget, build.CustomTarget]]':
```

Comment:
You don't need to quote the annotations because annotations are imported from future